BlueShift-iOS-SDK/InApps/BlueShiftInAppNotificationHelper.m

```diff
@@ -108,6 +108,13 @@ + (CGFloat)getPresentationAreaWidth {
     return presentationAreaWidth;
 }
 
++ (BOOL)checkAppDelegateWindowPresent {
+    if (![[UIApplication sharedApplication].delegate respondsToSelector:@selector(window)]) {
+        return NO;
+    }
+    return YES;
```

Comment:
can we inline this code like this? ignore if the syntax is not supported :) 

```return ![[UIApplication sharedApplication].delegate respondsToSelector:@selector(window)]```

Reply:
I have kept it like this for debugging and readability point of view. Also, `[[UIApplication sharedApplication].delegate respondsToSelector:@selector(window)]` returns an object so returning it directly wont help.